pylasu/model/errors.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from pylasu.model import Position, Node
+
+
+@dataclass
+class ErrorNode:
+    """An AST node that marks the presence of an error,
+     for example a syntactic or semantic error in the original tree."""
+    message: str = None
+    position: Optional[Position] = None
+
+
+@dataclass
+class GenericErrorNode(Node, ErrorNode):
```

Comment:
What is the difference between ErrorNode and GenericErrorNode?

Reply:
In Kolasu, ErrorNode is an interface and GenericErrorNode a concrete class. I ported the same hierarchy even though in Python we could just use ErrorNode.